docs/pages/develop/development-builds/expo-go-to-dev-build.mdx

```diff
@@ -0,0 +1,142 @@
+
+> After generating your **android** and/or **ios** directories, make sure you add them to **.gitignore** instead of checking them into Git. This ensures you can always regenerate the code locally or on CI when needed and never have to edit native code manually.
+
+### Option 1: Build on your local machine
+
+To build a native app on your local machine, follow the setup your environment guides for [Android](/workflow/android-studio-emulator/) and [iOS](/workflow/ios-simulator/) platforms. This involves setting up and configuring native build tools like Android Studio for Android and Xcode for iOS.
+
+Once you have everything set up, run the following command:
+
+<Tabs>
+
+<Tab label="Build for Android">
+
+<Terminal cmd={['$ npx expo run:android']} />
+
+</Tab>
+
+<Tab label="Build for iOS">
+
+<Terminal cmd={['$ npx expo run:ios']} />
+
+</Tab>
+</Tabs>
+
+By default, this will build and install the app on an Android Emulator/iOS Simulator. If you need to run the build on your phone, plug it into your computer (trust device and allow USB debugging if prompted, and on iOS enable [developer mode](https://docs.expo.dev/get-started/set-up-your-environment/?mode=development-build&buildEnv=local&platform=ios&device=physical#plug-in-your-device-via-usb-and-enable-developer-mode)) and run the above commands with the `--device` flag.
+
+### Option 2: Build on EAS
+
+Building on EAS servers is useful when:
+
+- you can't or don't want to set up your local development environment
+- you want to build an iOS app but don't own a Mac
+- you want to share the development builds with your team
+
+<BoxLink
+  title="Build on EAS"
+  description="How to create your Development Build on EAS"
+  href="/develop/development-builds/create-a-build/"
+  Icon={BookOpen02Icon}
+/>
+
+</Step>
+
+<Step label="2">
+
+## Start the bundler
+
+After building locally, `npx expo run:android:ios` will start the bundler automatically. But if you closed the bundler or are working on a dev client you built earlier, (re)start the Metro bundler with:
+
+<Terminal cmd={['$ npx expo start']} />
+
+When your project has `expo-dev-client` installed, the bundler will print out **Using development build**, and the QR code it shows will link into the development build you created instead of Expo Go.
+
+</Step>
+
+## Prebuild
+
+[**Prebuild** ](/workflow/continuous-native-generation/#prebuild) is a concept unique to Expo projects. It refers to the process of generating the **android** and **ios** directories based on your local configuration and properties.
+
+### When should you run prebuild
+
+You will need to run prebuild locally if you are building via `npx expo run:android|ios`, and change any native dependencies or configuration, such as:
+
+- Installing or updating a library containing native code
+- Changing `app.json` or `app.config.json`
```

Comment:
```suggestion
- Changing [app config](/workflow/configuration/)(`app.json`)
```

Let's link to the app config page and reference it this way as [per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-appjsonappconfigjsonappconfigjsappconfigts)